backend/manager/modules/utils/src/main/java/org/ovirt/engine/core/utils/ovf/OvfReader.java

```diff
@@ -167,6 +168,15 @@ protected void readDisk(XmlNode node, DiskImage image) {
         } else {
             image.setVolumeType(getDefaultVolumeType());
         }
+
+        if (node.attributes.get("ovf:backup") != null) {
+            if (!StringUtils.isEmpty(node.attributes.get("ovf:backup").getValue())) {
+                image.setBackup(DiskBackup.valueOf(node.attributes.get("ovf:backup").getValue()));
+            } else {
+                image.setBackup(DiskBackup.None);
```

Comment:
Don't you want to do the same for `node.attributes.get("ovf:backup") == null`?
Similar for `ovf:volume-format` or `ovf:volume-type`?

Reply:
ACK, didn't see that one